ut/linux_kernel/m0ut_main.c

```diff
@@ -34,8 +34,8 @@
  * marker is necessary to remove the warnings, keeping this blank to make
  * compiler happy.
  */
-MODULE_LICENSE();
 
+/* Added GPL per suggestions to avoid compilation error, may need to be reviewed */
```

Comment:
Move this comment close to the `MODULE_LICENSE()`.

Reply:
Why is this comment been left?

Reply:
Removed